test/functionalTest/cases/2750_common_metrics/2750_incoming_errors.test

```diff
@@ -295,25 +302,44 @@ Date: REGEX(.*)
                     "serviceTime": REGEX(.*)
                 },
                 "SP/get/metrics": {
-                    "incomingTransactionResponseSize": 337,
+                    "incomingTransactionResponseSize": 457,
```

Comment:
I guess this is due to the new `subservs` item within `sum` tenant...

Very interesting (and funny :) way to check that the modification has worked in the metric API system itself.